DOC.md

```diff
@@ -476,7 +385,63 @@ ooasp_special_cv(unique, (rack_frames, frame, frame_position)).
 
 The corresponding rules are defined in [ooasp/encodings/ooasp_check.lp](ooasp/encodings/ooasp_check.lp).
 
+----
+
+# Outdated Information
```

Comment:
Please call this "previous versions" and also add a link to the old version (release? tag?) for which this is relevant.